src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts

```diff
@@ -1,5 +1,7 @@
 // Name Request State interface
 export interface NameRequestIF {
+  legalType: string
```

Comment:
Could we use enum EntityTypes for this?
NB There's a (slow) move to use the same enum in FE and BE for this. I started something in Filings UI but it's incomplete overall. See https://github.com/bcgov/business-filings-ui/blob/master/src/enums/legalTypes.ts

Reply:
For sure, will apply the one we have for entityTypes 👍

Reply:
Sounds good.